Wabbajack.App.Blazor/App.xaml.cs

```diff
@@ -22,6 +24,7 @@ public App()
             .ConfigureServices(services => ConfigureServices(services))
             .Build()
             .Services;
+        _serviceProvider.GetRequiredService<SystemParametersConstructor>();
```

Comment:
Is this some leftover?

Reply:
This is so the assembly information (build, entry point, etc) actually gets logged at the program launch. There's probably a better way but this works so it's what I went with.